app/models/site_config.rb

```diff
@@ -28,6 +27,14 @@ class SiteConfig < RailsSettings::Base
     twitch: nil
   }
 
+  # Emails
+  field :email_addresses, type: :hash, default: {
+    default: "yo@dev.to",
+    business: "partners@dev.to",
+    privacy: "privacy@dev.to",
+    members: "members@dev.to"
```

Comment:
used only in events, probably this legacy feature will be disabled for other communities once we have feature toggles